mkdocs/docs/HPC/python.md

```diff
@@ -0,0 +1,164 @@
+Instead, we provide a large number of Python packages as modules that can be loaded into your environment.
+Once loaded, you can use the packages in your Python code.
+
+## Finding packages 
+
+If you need a particular Python package that is not part of the standard library, 
+you can search for it using the `module avail` command. 
+For example, to search for the `beautifulsoup` package, you can run:
+
+```bash
+$ module avail beautifulsoup
+BeautifulSoup/4.10.0-GCCcore-10.3.0
+...
+BeautifulSoup/4.12.2-GCCcore-13.2.0 (D)
+```
+
+This will show you all available versions of the `beautifulsoup` package.
+
+!!! Note
+    Not all Python packages can be found by searching for the package name. 
+    Some packages are installed as extensions of modules. 
+    For example, `numpy`, `scipy` and `pandas` are part of the `SciPy-bundle` module. 
+    You can use the following command to check which extensions are included in a module (if any):
+    
+    ```bash
+    module show <module_name>
+    ```
+
+If you cannot find a package, it could be that it is not available as a module.
+You can send us a [software installation request]({{ hpc_software_install }}), 
+or you can install the package in a [Python virtual environment](./setting_up_python_virtual_environments.md).
+
+## Loading packages
+
+If your python script does not use any third-party packages, you only have to load a Python module.
+If you need to use a third-party package, you will need to load the corresponding module.
+Loading such a module will also load a Python module, so you do not need to load both.
+
+For example, loading the `SciPy-bundle` module will also load the Python module:
+
+```bash
+$ module load SciPy-bundle/2023.11-gfbf-2023b
+$ module list
+Currently Loaded Modules:
+ ...
+ 21) Python/3.11.5-GCCcore-13.2.0
+ ...
+ 27) SciPy-bundle/2023.11-gfbf-2023b
+```
+
+## Example: Running Python scripts
+
+There are multiple ways to run Python scripts on the HPC systems. 
+You can start an interactive session on a cluster and run your script manually,
+or you can submit a job using a job script. 
+This section will show you how to run Python scripts using both methods on the following example script:
+
+```python title="script.py"
+import numpy as np
+
+random_numbers = np.random.rand(5)
+
+print(f"Random numbers: {random_numbers}")
+```
+
+First, we select a toolchain of the SciPy bundle module, which contains the `numpy` package, along with a Python module:
+
+```bash
+$ module avail SciPy-bundle
+```
+
+We will use the `SciPy-bundle/2023.11-gfbf-2023b` toolchain.
+Next, we decide on which cluster we want to run the script. 
+For this example, we will use the `donphan` cluster.
+
+To switch to the `donphan` cluster, we run:
+
+```bash
+$ module swap cluster/donphan
+```
+
+### Interactive session
+
+To run the script interactively, we need to start an interactive session on the cluster (`donphan` in this case):
```

Comment:
Like to page/section on interactive jobs?

Reply:
done in [link to interactive sessions page](https://github.com/hpcugent/vsc_user_docs/pull/689/commits/6f3ee91c57d4cec22783f7afdecde0fd8b11de47)